### ISSUE TITLE
Paramedics can't do their job now.

### ISSUE THREAD

**Issue by redguy999** [tgstation/tgstation#67498]
Reporting client version: 514.1584

## Round ID:
[184238](https://scrubby.melonmesa.com/round/184238) (It has been around for at least a few rounds now)

## Testmerges:
- [Adminwho now caches feedback links until admins are reloaded (and also works again)](https://www.github.com/tgstation/tgstation/pull/67307)
- [Optimizes Runechat](https://www.github.com/tgstation/tgstation/pull/65791)
- [Disables soft lock on traitor objectives](https://www.github.com/tgstation/tgstation/pull/67385)

## Reproduction:
1. Be a paramedic
2. Attempt to enter any department that isn't medical
3. notice you can't enter.

I don't see a PR removing paramedic's access to any of these departments (sans EVA), so I'm presuming this is a bug.

I can get removing Engineering access (though I don't like it), I can also understand removing cargo access, removing EVA is annoying, but science access basically only gave you access to nanites, and service basically only gave you access to the lathe, which doesn't mean much.

**Comment by Jolly-66** [comment]
Probably fallout from when Son of Space did access revamps and stuff

**Comment by Son-of-Space** [comment]
![image](https://user-images.githubusercontent.com/63861499/171776314-412eb2c6-d070-411b-b6eb-6235b1a10b40.png)

**Comment by Palpetinus** [comment]
This was last changed right here
https://github.com/tgstation/tgstation/pull/64969
Three months ago
Personal opinion: kinda killed paramedic for me. I always played the ol' search and rescue paramed, keeping the crew monitor on another screen, pinpointer (now LifeLine) open and in a pocket (now ID slot) and so on.
Not being able to go into engineering to claw the SM-Rads infested engineers out of there and similar such things, kinda ruined it for me. Oh well.

**Comment by JohnFulpWillard** [comment]
@Palpetinus 
That PR is not the fault of it
The new ID changes reworked what has access to what.

Following screenshots are from Deltastation, before and after the refactor

```Cargo```
Mining access was enough to enter Cargo bay, that is not the case anymore. (Shaft Miners were instead given Cargo shuttle access so they aren't affected, which is a problem too because they're only supposed to have it on lowpop).
Old:
![image](https://user-images.githubusercontent.com/53777086/171850219-990a2984-d1b6-448d-9f5b-3e58bb489b4e.png)
New:
![image](https://user-images.githubusercontent.com/53777086/171850433-948fe5a8-b5b6-481f-8185-9b84fa479703.png)
![image](https://user-images.githubusercontent.com/53777086/171850456-55a5c98b-8a72-4de7-9814-15b9dd2b7835.png)

``Science``
Paramedics have Research access, but not Science access, which is what was needed for the doors now.
Old:
![image](https://user-images.githubusercontent.com/53777086/171850039-f5034798-d76c-42dd-8452-4c2a0b8c3e66.png)
![image](https://user-images.githubusercontent.com/53777086/171850047-9b0c9bb2-2923-4352-bdf9-edf7558df076.png)
New:
![image](https://user-images.githubusercontent.com/53777086/171850363-65e61c91-98cf-4a37-b8a3-e50dc8f57022.png)
![image](https://user-images.githubusercontent.com/53777086/171850403-4b27a863-7cf1-416a-9b68-5e85c3b65f6c.png)

``Engineering``
They have Construction, not Engineering. In the past Construction was needed to enter Engineering front doors (it also removed Command's access to these areas at the same time).
Old: 
![image](https://user-images.githubusercontent.com/53777086/171849991-5858bc29-f4ae-4a26-aa5a-0b460021e130.png)
![image](https://user-images.githubusercontent.com/53777086/171850077-78cdbde0-c4f0-4c71-b1dc-073f49a48f0f.png)
![image](https://user-images.githubusercontent.com/53777086/171850145-8f368f2f-2925-40b7-9712-f2f0a1a1f4eb.png)
New:
![image](https://user-images.githubusercontent.com/53777086/171850505-1792491e-d586-40ed-9cd4-23e8a0205a3f.png)
![image](https://user-images.githubusercontent.com/53777086/171850530-0fc5aec6-7df4-4744-b251-d4454e9429fc.png)

TL;DR Don't blame my nerf, it wasn't me.

**Comment by Son-of-Space** [comment]
With John's comment, it makes the issue a little more complicated. General access is there to be the generalized access for members of that department. In order to make it so paramedics can have access to the front doors on a department, we would need to add a snowflake access specifically for paramedics and add it on the front doors of every department across all stations. That seems to me like it wouldn't be ideal especially since part of what I've been trying to do with the ID reworks is get to a place where we can SIMPLIFY our accesses rather than make them more complicated. We can't just give paramedics general access to all of the departments, either, for the same reasons that John put out in the PR you referenced. I will have to talk more with him on it and see if we can come up with a satisfying solution.

**Comment by Mothblocks** [comment]
@Dorsisdwarf Which is it? "Not a bug" should only be closed issues, if this is unintentional then please remove it and just keep oversight
